.scalafmt.conf

```
@@ -1,5 +1,5 @@
 version = 3.9.8
-runner.dialect = scala213
+runner.dialect = scala212source3
```

```suggestion
runner.dialect = scala213source3
```
shouldn't it be this by default? We might need to do scala213source3 for .sbt and things inside project/